### ISSUE TITLE
Public Beta List

### ISSUE THREAD

**Issue by klappy** [unfoldingWord/translationCore#444]
As a development team, I want to know what all has to be completed before we can release our Public Beta.

### Refactoring the tools to use semi-redux
- [x] Translation Words components
- [x] Scripture Pane
- [x] Comment box
- [x] Proposed Changes
- [x] Package Manager
- [x] Upload Modal
- [x] Login
- [x] Modals
- [x] Menu Refactored

### Core App
- [x] Menu selector bar centered
- [x] Git functions working well
- [x] Updated UI
- [x] Loading projects
- [x] Modal Preferences/Settings

### Tools
- [x] Git submodules for tools
- [x] One working Tool (Proper Nouns)
- [x] UI
- [x] Sliding Resource Pane

### Stretch goal
- [ ] QA Bake

### After Beta
- [ ] Package manager refinements

**Comment by mannycolon** [comment]
@klappy I believe **Git submodules for tools** should be good to go. I just merged it in with develop
